client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -45,6 +45,9 @@
 
 @six.add_metaclass(abc.ABCMeta)
 class _DeployableEntity(_ModelDBEntity):
+
+    _MODEL_KEY = None
```

Comment:
suggestion: if `None` isn't a valid `_MODEL_KEY` we should convert this to a property which raises a `NotImplementedError` if the sub-class doesn't supply a `_MODEL_KEY`.

Reply:
I had considered this, but I had decided against it since I would like `_MODEL_KEY` to be accessible from the `class` object, not just on instances.

Although maybe that's fine for how it's used in the client? I'll take another look through the code.